aggregator/src/app/BundleTable.ts

```diff
@@ -53,25 +61,38 @@ export type BundleRow = Row;
 
 const tableOptions: TableOptions = {
   id: { type: DataType.Serial, constraint: Constraint.PrimaryKey },
+  status: { type: DataType.VarChar },
   hash: { type: DataType.VarChar },
   bundle: { type: DataType.VarChar },
   submitError: { type: DataType.VarChar, nullable: true },
   eligibleAfter: { type: DataType.VarChar },
   nextEligibilityDelay: { type: DataType.VarChar },
+  receipt: { type: DataType.VarChar },
 };
 
 function fromRawRow(rawRow: RawRow): Row {
-  const parseResult = parseBundleDto(JSON.parse(rawRow.bundle));
-  if ("failures" in parseResult) {
-    throw new Error(parseResult.failures.join("\n"));
+  const parseBundleResult = parseBundleDto(JSON.parse(rawRow.bundle));
+  if ("failures" in parseBundleResult) {
+    throw new Error(parseBundleResult.failures.join("\n"));
+  }
+
+  const status = rawRow.status;
+  if (!isValidStatus(status)) {
+    throw new Error(`Not a valid bundle status: ${status}`);
```

Comment:
Does this error or `isValidStatus` need a test?

Reply:
I added `isValidStatus()` more for the typing benefits rather than to check if the DB returns a valid status. The DB is just returning the status as a string and I want it to be typed as a BundleStatus which is why I needed to add this.  I also added the error for typing reasons.  I don't expect this error to be thrown though.  I'd lean towards saying this doesn't need a test but I'd be happy to add one

Reply:
Ok nice, how it is already sounds good